### ISSUE TITLE
Improve handling of non-conforming Export file names

### ISSUE THREAD

**Issue by rwdudley-usgs** [USGS-R/wateRuse#186]
There have been several instances of users unable to load their export files because their filename format doesn't conform to the format the loading function wants to see. Does AWUDS output the correct default format? If so, are AWUDS users overriding the format? If not, are users free to make up any filename format they want? Depending on the answers, we might want to consider writing in some error catching to inform the user to fix their filename formats or expand the ability to read other filename formats or some other solution.

**Comment by grrmartin-USGS** [comment]
The new AWUDS 3.2.4 default export file names work OK in wateRuse 1.0.6. (even though they changed the format of the County AWUDS export-file name for some reason. The other export files (Aquifer and HUC8) did not have a change in the file name format. ).  Anyway, the default formats work ok.  Users can revise export file names, but only by appending a string to the file name.  (wateRuse gets the YEAR from the export file name.)

**Comment by grrmartin-USGS** [comment]
Was this a frequent failure for user's loading data?  I think the current AWUDS 3.2.4 default names work ok.  I think users just need to only customize the file names by appending to the default names. Correct? wateRuse does read the year of the data from the file name.

**Comment by rwdudley-usgs** [comment]
At present, the only thing being parsed out of the file name is the year with this code:
`year<-regmatches(file_open,regexpr('[1,2][0,9][0-9][0-5]',file_open))`
So as long as users don't change the year or add another string to the filename that looks like a plausible year (19xx, or 20xx) it should be fine. Are people still having issues with this? Does anyone have examples we can test?

**Comment by jshourds-usgs** [comment]
I don't think this is an issue anymore. In our training and instruction manual, we discourage people from changing the name of the AWUDS export file.  We do tell them that it's okay if they add a string of characters at the end, after the year, if they need to distinguish their AWUDS exports from one another. 

I'm wondering if down the line, we might want to change the end of the code to allow for other years as it looks like it will only allow for years ending in 0-5.  Compilation years end in 0 or 5 so I think that's why that was built in but down the line we may want to allow for other years, especially as we look into adding site-specific functionalities.  Just food for thought.

**Comment by rwdudley-usgs** [comment]
Okay, I'll close this enhancement then. 
I agree about allowing for other years down the road. We can change it pretty easily when it comes to that - in the meantime I can see value in leaving it as is because it makes the code just a bit "smarter" in reading the year correctly from the filename.

**Comment by grrmartin-USGS** [comment]
Encountered glitch for new 2016 data: years ending 6-9 are not read at present. 
Rob informs:
From that issue 186 link you sent I see the following:
year<-regmatches(file_open,regexpr('[1,2][0,9][0-9][0-5]',file_open))
So it only allows years ending in 0, 1, 2, 3, 4, or 5.
You could change it to:
year<-regmatches(file_open,regexpr('[1,2][0,9][0-9][0-9]',file_open))

**Comment by grrmartin-USGS** [comment]
Will attempt code revision, tests, and pull-request.
